### ISSUE TITLE
Authentication failed

### ISSUE THREAD

**Issue by tlian** [CiscoDevNet/FTDAnsible#116]
The module failed to authenticate against FTD. 
Tested out with both latest code on master branch and v0.3.0. 

**Device info**
Model: Cisco Firepower Management Center 4500
Software Versions: 6.3.0.3

Here is ansible play
`- hosts: ftd
  connection: httpapi
  tasks:
    - name: Get existing networks
      ftd_configuration:
        operation: getNetworkObjectList
        register_as: my_networks`

Here is inventory 
`[ftd]
192.XX.XXX.XXX ansible_network_os=ftd ansible_user="myusername" ansible_password="mypassword" ansible_httpapi_port=443 ansible_httpapi_use_ssl=True ansible_httpapi_validate_certs=False`

**Error message**
fatal: [192.XX.XXX.XXX]: FAILED! => {"msg": "Ansible could not determine the API token path automatically. Please, specify the 'ansible_httpapi_ftd_token_path' variable in the inventory file."}

Since the above error, tried out the following by explicitly specifying FTD token path (with v1,v2 and v3).
Inventory file looked like below
`[ftd]
192.XX.XXX.XXX ansible_network_os=ftd ansible_user="myusername" ansible_password="mypassword" ansible_httpapi_port=443 ansible_httpapi_use_ssl=True ansible_httpapi_validate_certs=False ansible_httpapi_ftd_token_path=/api/fdm/v2/fdm/token`

**Error message**
`fatal: [192.XX.XXX.XXX]: FAILED! => {"msg": "The API token path is incorrect. Please, check correctness of the 'ansible_httpapi_ftd_token_path' variable in the inventory file."}`

**Comment by sdbrain** [comment]
@tlian the ansible modules work only with the ftd rest API. It does not yet work with FMC(firepower management center).

**Comment by dalamanster** [comment]
Hello community, is there any plan for creating ansible collections for FMC? 
If we are managing FTDs via FMC, it is not possible to use this FTD ansible collection (only FMC can manage FTD device).

Thank you

**Comment by dhemwani** [comment]
> Hello community, is there any plan for creating ansible collections for FMC?
> If we are managing FTDs via FMC, it is not possible to use this FTD ansible collection (only FMC can manage FTD device).
> 
> Thank you

No, we cant use the FTD API Ansible collection for FMC managed devices. We dont have an Ansible collection available for FMC API at the moment. It's on the roadmap for future.
